src/redux/reducers/articleReducer.js

```diff
@@ -14,6 +21,17 @@ export default (state = initialState, action) => {
       return { ...state, articles: action.data };
     case GET_TRENDING_ARTICLES:
       return { ...state, trendingArticles: action.data };
+    case SAVE_ARTICLE_DATA:
+      return { ...state, article: action.payload };
+    case EDIT_ARTICLE:
+      return { ...state, article: action.payload };
+    case DELETE_ARTICLE:
+      return {
```

Comment:
@chingsley what are we doing here?

Reply:
You said I should redirect to the landing page upon delete, instead of rendering the success message on the modal. This is how I communicate the delete action from the store to the landing page so that the success message will be rendered there.